clusters/app.ci/cert-manager/registry-ci-openshift-org_certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    gcp-project: openshift-ci-infra
+  name: registry-ci-openshift-org
+  namespace: openshift-image-registry
+spec:
+  secretName: public-route-tls
```

Comment:
Maybe name it like the cert but with a `-tls` suffix? The build-in route is also public